codegen/smithy-ruby-rails-codegen/middleware/request_id.rb

```diff
@@ -0,0 +1,16 @@
+module Middleware
```

Comment:
This might be global scope isn't it? I think we'd need to write out Service::Middleware

Reply:
Yeah, this either needs to be in a core gem or needs to be codegened out with a unique namespace....